packages/server/migrations/20230914192911_add-archive-date-to-grants.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+    // , ALTER COLUMN close_date TYPE timestamp USING close_date::timestamp, ADD COLUMN archive_date TYPE timestamp
+    return knex.raw(`ALTER TABLE grants ALTER COLUMN open_date TYPE timestamp USING open_date::timestamp,
+    ALTER COLUMN close_date TYPE timestamp USING close_date::timestamp,
+    ADD COLUMN archive_date timestamp`);
```

Comment:
I'd suggest using `date` as the new type for these columns:
```suggestion
    return knex.raw(`ALTER TABLE grants ALTER COLUMN open_date TYPE date USING open_date::date,
    ALTER COLUMN close_date TYPE date USING close_date::date,
    ADD COLUMN archive_date date`);
```

Note: These can (also) compare to timestamps like `now()`, so it should be perfectly compatible with our use-case, e.g. 
```sql
select '2023-09-15'::date < now() as is_before;
 is_before 
-----------
 t
(1 row)

select now()::date < now() as is_before;  -- Compared to a timestamp, date represents "midnight"
 is_before 
-----------
 t
(1 row)

select '2023-09-15'::date as some_date;
 some_date  
------------
 2023-09-15
(1 row)

select '2023-09-15'::date::timestamp as some_timestamp;  -- Easily converted to timestamp if needed
      some_date      
---------------------
 2023-09-15 00:00:00
(1 row)

select '2023-09-15'::date::text as some_text;  -- Compatible with the downward migration
 some_text  
------------
 2023-09-15
```

Reply:
Done 👍